pkg/virt-launcher/virtwrap/nichotplug.go

```diff
@@ -28,75 +28,59 @@ import (
 	"kubevirt.io/client-go/log"
 
 	"kubevirt.io/kubevirt/pkg/network/cache"
+	"kubevirt.io/kubevirt/pkg/network/domainspec"
 	netsetup "kubevirt.io/kubevirt/pkg/network/setup"
 	netvmispec "kubevirt.io/kubevirt/pkg/network/vmispec"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/cli"
 )
 
-func hotplugVirtioInterface(vmi *v1.VirtualMachineInstance, dom cli.VirDomain, currentDomain *api.Domain, updatedDomain *api.Domain) error {
+func hotplugVirtioInterface(vmi *v1.VirtualMachineInstance, dom cli.VirDomain, domainInterfaces []api.Interface, domain *api.Domain) error {
```

Comment:
IMO, this makes a poor API. 

Either you pass the old and new domains, or the old and new domain interface slices.